crypto/cipher/aes_gcm_ossl.c

```diff
@@ -439,7 +460,7 @@ static const srtp_cipher_test_case_t srtp_aes_gcm_test_case_0a = {
     srtp_aes_gcm_test_case_0_ciphertext, /* ciphertext  + tag        */
     20,                                  /* octets in AAD            */
     srtp_aes_gcm_test_case_0_aad,        /* AAD                      */
-    GCM_AUTH_TAG_LEN_8,
+    GCM_AUTH_TAG_LEN_8,                  /* */
```

Comment:
I saw a few empty comment lines.  Did clang-format add those or did you?

Reply:
I added them in order that clang format does not put multiple assignments on the same line.
Seamed like the lesser evil in order to keep some of the existing formatting.